rke/templates/coredns_v1.8.3-ca-probes.go

```diff
@@ -0,0 +1,354 @@
+package templates
+
+const CoreDNSTemplateV183CAProbes = `
```

Comment:
Could we rename this to `CoreDNSTemplateV183Rancher2`? And file name would be `coredns_v1.8.3-rancher2`? 

To add context, we don't have a convention for this yet so there are a few templates named the same way. But as we're introducing more of these, I think everyone will name it their own way introducing inconsistency and it'll be harder to track the order (for eg, v183CAProbes, v183Privileged..etc). 

I suggest we use the same convention as we do for system images - adding suffix `rancherX` and incrementing X with each update. And we add a Rancher changelog to list the specific changes so it's easier to track them in future. 

Example:  https://github.com/rancher/kontainer-driver-metadata/pull/793/commits/3c904c5a0a9e49d3ef6756a6a3b34377aa043b88

Reply:
Yep, I did another PR review this morning that had a `Rancher1` appended to the end. Can do. I'm also going to update my template description